lib/block-supports/layout.php

```diff
@@ -402,7 +402,7 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 
 	$global_settings        = gutenberg_get_global_settings();
 	$global_layout_settings = _wp_array_get( $global_settings, array( 'layout' ), null );
-	$used_layout            = isset( $block['attrs']['layout'] ) ? $block['attrs']['layout'] : _wp_array_get( $block_type->supports, array( '__experimentalLayout', 'default' ), array() );
+	$used_layout            = isset( $block['attrs']['layout'] ) ? $block['attrs']['layout'] : _wp_array_get( $block_type->supports, array( 'layout', 'default' ), array() );
```

Comment:
The above two changes fall back to `__experimentalLayout` if `layout` isn't available. Should we do that for this one, too?

Reply:
I didn't change it because it seems to be accessing the correct value anyway, although yeah it might be best to be on the safe side.